l10n_fr_state/migrations/9.0.0.1.1/pre-migration.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+# © 2015-2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    # key : source region XMLID - value : destination region XMLID
+    new_region_map = {
+        'res_country_state_rhonealpes': 'res_country_state_auvergne',
```

Comment:
It should be great to have correct name for xml_id. For exemple, res_country_state_auvergne_rhone_alpes. It is less error prone.
You can make it easy with

```
from openupgradelib import openupgrade

xmlid_renames = [
     ('res_country_state_auvergne', 'res_country_state_auvergne_rhone_alpes'),
]

@openupgrade.migrate()
def migrate(cr, version):
    openupgrade.rename_xmlids(cr, xmlid_renames)
```
in the premigration script.